api/models/schema.py

```diff
@@ -12,549 +12,580 @@
-        field_schema.update(type='string', title='Other', description="Please specify another language.")
+  @classmethod
+  def __modify_schema__(cls, field_schema: Dict[str, Any]) -> None:
+    field_schema.update(type='string', title='Other',
+                        description="Please specify another language.")
 
 
 class IdentifierStr(str):
-    @classmethod
-    def __modify_schema__(cls, field_schema: Dict[str, Any]) -> None:
-        field_schema.update(type='string', title='Identifier')
+  @classmethod
+  def __modify_schema__(cls, field_schema: Dict[str, Any]) -> None:
+    field_schema.update(type='string', title='Identifier')
 
 
 class Grant(SchemaBaseModel):
-    type: str = Field(
-        alias="@type",
-        default="MonetaryGrant",
-        description="This metadata represents details about a grant or financial assistance provided to an "
-                    "individual(s) or organization(s) for supporting the work related to the record."
-    )
-    name: str = Field(
-        title="Name or title",
-        description="A text string indicating the name or title of the grant or financial assistance.")
-    description: Optional[str] = Field(description="A text string describing the grant or financial assistance.")
-    identifier: Optional[str] = Field(
-        title="Funding identifier",
-        description="Grant award number or other identifier."
-    )
-    funder: Optional[FunderOrganization] = Field(
-        description="The organization that provided the funding or sponsorship."
-    )
+  type: str = Field(
+      alias="@type",
+      default="MonetaryGrant",
+      description="This metadata represents details about a grant or financial assistance provided to an "
+                  "individual(s) or organization(s) for supporting the work related to the record."
+  )
+  name: str = Field(
+      title="Name or title",
+      description="A text string indicating the name or title of the grant or financial assistance.")
+  description: Optional[str] = Field(
+    description="A text string describing the grant or financial assistance.")
+  identifier: Optional[str] = Field(
+      title="Funding identifier",
+      description="Grant award number or other identifier."
+  )
+  funder: Optional[FunderOrganization] = Field(
+      description="The organization that provided the funding or sponsorship."
+  )
 
 
 class TemporalCoverage(SchemaBaseModel):
-    startDate: datetime = Field(
-        title="Start date",
-        description="A date/time object containing the instant corresponding to the commencement of the time "
-                    "interval (ISO8601 formatted date - YYYY-MM-DDTHH:MM).",
-        # TODO: these are failing due to a problem with transpiled dependencies inside cznet-vue-core
-        # formatMaximum={"$data": "1/endDate"},
-        # errorMessage= { "formatMaximum": "must be lesser than or equal to End date" }
+  startDate: datetime = Field(
```

Comment:
@pkdash - All other changes in this file are from linter, these are the only real changes.

Reply:
What linter are you using? I think the standard indention is 4 spaces which it had originally. Now it seems using 2 spaces for indentation.

Reply:
I'm using autopep8

`
"[python]": {
      "editor.formatOnType": true,
      "editor.defaultFormatter": "ms-python.autopep8"
    },
`

Reply:
I have undone the indentations now.